src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManager.java

```diff
@@ -348,6 +348,19 @@ public GameboardDTO getLiteGameboard(final String gameboardId) throws SegueDatab
         return this.gameboardPersistenceManager.getLiteGameboardById(gameboardId);
     }
 
+    /**
+     * Get a list of gameboards by their ids.
+     *
+     * @param gameboardId
+     *            - the ids to find.
+     * @return a list of lite gameboards without their question data fully expanded.
+     * @throws SegueDatabaseException
+     *             - a database error has occurred.
+     */
+    public List<GameboardDTO> getLiteGameboards(final Collection<String> gameboardId) throws SegueDatabaseException {
```

Comment:
Or, even better, if we eventually want to move towards a progressive enhancement approach rather than progressive destruction (…) approach[^1], to make a new query at the persistence level that just doesn't query unnecessary data, and returns a collection of `LiteGamboardDO`s instead?

[^1]: This is only tangentially related to this approach, but once you pick an idiom, it's a good idea to not stray too far away from it.